docs/tutorials/tabular_prediction/tabular-gpu.md

```diff
@@ -41,19 +41,47 @@ hyperparameters
 
 ### Text models
 
-Text model preset to use can be set via:
+Text model's configuration can be set via:
 
 ```{.python}
-hyperparameters['AG_TEXT_NN'] = ['<preset>']
+hyperparameters['AG_TEXT_NN'] = {
+    "model": "/path/to/model/yaml/file",
+    "data": "/path/to/data/yaml/file",
+    "optimization": "/path/to/optimization/yaml/file",
+    "environment": "/path/to/optimization/yaml/file",
+}
 ```
 
-Available text model presets:
+
+Text model uses the following configurations as default.
```

Comment:
We may ask the user to refer to another document page about how to customize the text predictor and multimodal predictor.

Reply:
Agree. We can change the customization.md tutorial to do this.